providers/apis/polymarket/config.go

```diff
@@ -0,0 +1,18 @@
+package polymarket
+
+import (
+	"time"
+
+	"github.com/skip-mev/slinky/oracle/config"
+)
+
+var DefaultAPIConfig = config.APIConfig{
+	Name:             Name,
+	Atomic:           true,
+	Enabled:          true,
+	Timeout:          3 * time.Second,
+	Interval:         500 * time.Millisecond,
```

Comment:
does the block time of polygon need to be considered here? do we have any info on the expected rate limits for hitting their clob endpoint?

Reply:
their team told me the following for rate limits:

In general you can't do more than 20req/s
You can't request orders and trades combined more than 20req/s
You can't do mora than 10 reqs to /book per second
and you can't do more than 5 reqs to /books per second

as for polygon block time; their orderbook is offchain, and thats what the CLOB API speaks to. so no need to consider the polygon blocktime

Reply:
would you might adding this to the docs just so we can tract it in case we need it.

Reply:
added[rate limit docs](https://github.com/skip-mev/slinky/pull/629/commits/a7d24df7dbb64fa790fcf442259c55bc858f1296)